pulp_docker/app/serializers.py

```diff
@@ -290,3 +290,49 @@ class DockerRecursiveAddSerializer(serializers.Serializer):
         write_only=True,
         required=False
     )
+
+
+class TagCopySerializer(serializers.Serializer):
+    """
+    Serializer for copying tags from a source repository to a destination repository.
+    """
+
+    source_repository = serializers.HyperlinkedRelatedField(
```

Comment:
@asmacdo do you think you can add also source_repo_version?

Reply:
Sure